### DISCUSSION TITLE
github support does not response at all. it's been a week.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#77141] [Other Feature Feedback, Questions, & Ideas]
github support does not response at all. it's been a week.
![image](https://github.com/community/community/assets/151508256/04bc3337-8fd1-4700-be31-caea34a182ba)

**Comment by ghost** [comment]
**no response !!!**

**Comment by queenofcorgis** [comment]
Hi @dDDDDDDaniel, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Reply by ghost** [reply]
how long should i wait ??
